features/step_definitions/files_steps.rb

```diff
@@ -11,11 +16,21 @@
 
 
 
+Then(/^I (?:still|again) have my uncommitted file$/) do
+  verify_uncommitted_file file_name: @uncommitted_file_name
+end
+
+
 Then(/^(?:now I|I still) have the following committed files$/) do |files_data|
   files_data.diff! files_in_branches
 end
 
 
+Then(/^I don't have my uncommitted file$/) do
```

Comment:
Looking at this again, this is a chance to improve the name of this step definition as well. It was never that great to begin with. "I don't have my uncommitted file" is technically not true, its still there, just stashed away. What do you think about `Then my uncommitted file is( still)? stashed away`?